admin_manual/installation/system_requirements.rst

```diff
@@ -23,9 +23,9 @@ For best performance, stability and functionality we have documented some recomm
 |                  | - openSUSE Leap 15.5                                                  |
 |                  | - CentOS Stream                                                       |
 +------------------+-----------------------------------------------------------------------+
-| Database         | - **MySQL 8.0+** or MariaDB 10.3/10.4/10.5/**10.6** (recommended)     |
+| Database         | - **MySQL 8.0+** or MariaDB 10.3/10.5/**10.6** (recommended)/10.11    |
```

Comment:
We run CI against [10.2](https://github.com/nextcloud/server/blob/stable28/.drone.yml#L275) and [10.4](https://github.com/nextcloud/server/blob/stable28/.drone.yml#L313) but not against 10.11
So should drop 10.11 from the list here until it's tested.
And when there is no reason to test 10.4 and 10.2 we should remove them from CI and queue 10.3 and 10.5 instead?